### PR TITLE
pallet treasury & asset-rate initial config

### PR DESCRIPTION
**Pull request by MrishoLukamba**
Some config I need suggestion on what they should be

1. spending period
2. payout period

### CODE REVIEW

**Comment by Wizdave97** [comment]
We can start conservatively by using polkadot's config for those values.

**Comment by Wizdave97** [comment]
pallet-collectives is missing.

**Comment by Wizdave97** [comment]
Your branch does not have the latest changes from main.

**Comment by MrishoLukamba** [comment]
> Your branch does not have the latest changes from main, that's why some integration tests are failing.

rebased

File: parachain/runtimes/gargantua/src/lib.rs [hunk 1]
```diff
@@ -553,6 +568,95 @@ impl pallet_utility::Config for Runtime {
...
+/// A way to pay from treasury
+impl pallet_treasury::Config for Runtime {
+	type Currency = Balances;
+	type RejectOrigin = EnsureRoot<AccountId32>;
+	type RuntimeEvent = RuntimeEvent;
+	type SpendPeriod = SpendingPeriod;
+	type Burn = ();
+	type PalletId = TreasuryPalletId;
+	type BurnDestination = ();
+	type WeightInfo = ();
```
Comment (Wizdave97) [inline parachain/runtimes/gargantua/src/lib.rs:622]: Run the benchmarks for the new pallets.

**Pull request merged**